scripts/rollup/modules.js

```diff
@@ -171,13 +171,6 @@ function getInternalModules() {
   };
 }
 
-function replaceInternalModules() {
-  // we inline these modules in the bundles rather than leave them as external
-  return {
-    "'react-dom'": `'${resolve('./src/renderers/dom/ReactDOM.js')}'`,
```

Comment:
Rather than have this special case in all bundles, I opted for importing `ReactDOM` explicitly in test utils.